infra/onert-micro/CMakeLists.txt

```diff
@@ -98,4 +98,6 @@ if(ENABLE_COVERAGE)
     target_link_libraries(onert_micro_coverage INTERFACE gcov)
 endif(ENABLE_COVERAGE)
 
-add_subdirectory("${NNAS_PROJECT_SOURCE_DIR}/onert-micro" "${CMAKE_BINARY_DIR}/onert-micro")
+if(NOT DISABLE_ONERT_MICRO_DIR)
```

Comment:
This part is needed to break `add_subdirectory` loop:
`infra/onert-micro/CMakeLists.txt` -> `onert-micro/CMakeLists.txt` -> `onert-micro/standalone/CMakeLists.txt` -> `infra/onert-micro/CMakeLists.txt`

I think, it is better to move some code from this file into separate files in infra.

For example `infra/onert-micro/cmake/utils.cmake` and/or `infra/onert-micro/cmake/variables.cmake` and include these files from `infra/onert-micro/CMakeLists.txt` and `infra/onert-micro/standalone/CMakeLists.txt`

Reply:
Done